content/WebAuthn/Supporting_U2F_or_FIDO2_Security_Keys_on_iOS_or_iPadOS/Known_Issues.adoc

```diff
@@ -0,0 +1,33 @@
+== Known Issues (when using Safari)
+
+*(Updated Dec 20, 2019) This known issues section will be updated as issues are reported.
+
+=== 1. User Verification (PIN Support)
+
+Whether a FIDO2 PIN is set, required, or preferred, the Safari browser dialog does not prompt for a PIN.
+
+[NOTE]
+======
+If the service from which the user is authenticating is a U2F flow only, then PIN on FIDO2 is a non-issue and the user will successfully authenticate. Learn more in the link:../WebAuthn_Developer_Guide/User_Presence_vs_User_Verification.adoc[User Presence vs User Verification section of our WebAuthn Developer Guide].
+======
+
+image:u-v.png[]
```

Comment:
this image contains a yellow shadow over the first column, I don't think that's meant o be present. Also maybe these table should be tables in the document rather than included images with text?

Reply:
There's a strong push here for lots of graphic content ("graphic" in the sense of illustrative). In my view, it is very problematic, not only because it is often difficult to read the text in images, but also because it is such a nuisance to update the information in a graphic. As with table 1, I will change it out for a table if I have time.

Reply:
I don't think this is a very wise decision.
Having content like this (simple table) be graphic means problems for searchability, accessibility and a (potentially) worse experience on mobile.

Reply:
I've substituted tables for the graphics